crates/pop-parachains/src/utils/git.rs

```diff
@@ -261,6 +258,7 @@ impl GitHub {
 			.ok_or(Error::Git("the repository name is missing from the github url".to_string()))?)
 	}
 
+	#[cfg(test)]
 	pub(crate) fn release(repo: &Url, tag: &str, artifact: &str) -> String {
```

Comment:
It is being used by some tests still and the quickest way to resolve the build warning was to just add the attribute.

I figure that this might be moved into a `common` crate at some point so could be dealt with then, but happy to address now by moving it to tests if you'd prefer?

Reply:
Yes, we have a task to move this functionality into a `common` crate: https://github.com/r0gue-io/pop-cli/issues/124. This can be tackled later.